demo/blocks_plug/run.py

```diff
@@ -0,0 +1,30 @@
+import gradio as gr
+
+text_identity, text_input, text_output = gr.Blocks(), gr.Blocks(), gr.Blocks()
```

Comment:
I think we can rename inner demos better to clarify the message.
```suggestion
identity_demo, input_demo, output_demo= gr.Blocks(), gr.Blocks(), gr.Blocks()
```

Reply:
Sounds good, will do